.JuliaFormatter.toml

```diff
@@ -0,0 +1,8 @@
+style = "sciml"
+
+margin = 92
+indent = 4
+yas_style_nesting = true
+
+ignore = ["src/equations/shallow_water_two_layer_1d.jl",
+          "src/equations/shallow_water_two_layer_2d.jl"]
```

Comment:
In TrixiParticles, we're using `whitespace_in_kwargs = false` because that seems to be more common in Julia. It's also more consistent with the style of Trixi so far.

Reply:
Right. I don't have a strong opinion here either, thus leaning towards the default for the SciML style. IIRC, @ranocha prefers the whitespace.

Reply:
Well, I am getting used to it since SciML changed :sweat_smile: Both would be fine with me but I think the argument "least amount of changes" could be used here as well